app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class ReviewsController < ApplicationController
+  before_action :find_film
+  decorates_assigned :film
+
+  def create
+    @review = @film.reviews.create(review_params)
+    @review.user_id = current_user.id
+    if @review.save
```

Comment:
Now you do 2 operations on database, you should improve it by changing `create` to `new`.

Also - why you use `render 'films/show'`? Why not `new` action? Please read a bit about CRUD - you did it totally wrong.